client/public/loginPage/loginScript.js

```diff
@@ -0,0 +1,45 @@
+const loginForm = document.getElementById("loginFrm");
+const emailInpt = document.getElementById("emailInpt");
+const passwordInpt = document.getElementById("passwordInpt");
+
+loginForm.addEventListener("submit", async (e) => {
+  e.preventDefault();
+  const userInfo = {
+    email: loginForm.email.value,
+    password: loginForm.password.value,
+  };
+  try {
+    if (userInfo.email !== null || userInfo.password !== null) {
+      const data = await login(userInfo);
+      if (data.isLoggedIn) {
+        if (data.role === "manager") {
+          window.location.replace("http://localhost:4000/public/dashboard");
+        }
+        if (data.role === "employee") {
+          window.location.replace("http://localhost:4000/public/dashboard");
+        }
+      } else {
+        errorMsg.style.display = "block";
```

Comment:
avoid mixing layer (style and js)

Reply:
loginScripts.js is a frontend file that manages the behavior of the html file. whay can't I use it to modify html elements?

Reply:
When we'll get to React you'll understand the separation of concern idea better. In general you should avoid changing the css from the JS file, you can simply create a css class and assign it to the element when upon an if statement.